<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkillsChooser</class>
 <widget class="QWidget" name="SkillsChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Available skills:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Skill:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="skillsSelector"/>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Level:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="levelsSelector"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Drag to node:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addSkill">
       <property name="text">
        <string>Add skill</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeSkill">
       <property name="text">
        <string>Remove skill</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_12">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Level description:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="levelDescription">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SkillPack:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="skillPackFile">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
